scripts/import_cldr.py

```diff
@@ -915,10 +915,6 @@ def parse_currency_formats(data, tree):
             curr_length_type = length_elem.attrib.get('type')
             for elem in length_elem.findall('currencyFormat'):
                 type = elem.attrib.get('type')
-                if curr_length_type:
-                    # Handle `<currencyFormatLength type="short">`, etc.
-                    # TODO(3.x): use nested dicts instead of colon-separated madness
```

Comment:
In hindsight the `(3.x)` in the TODO was probably too conservative. After all we've always noted the format dicts may change between version.

👍

Reply:
Yeah, and also as mentioned, the short format was always returning the format for 100 trillions, so it's probably safe to assume it's not actively in use